xarray/backends/api.py

```diff
@@ -233,6 +233,101 @@ def maybe_decode_store(store, lock=False):
     return maybe_decode_store(store)
 
 
+def open_dataarray(*args, **kwargs):
```

Comment:
as long as we're copying the docstring, let's copy the arguments for the function signature, too -- it helps auto-complete.